cmd/snap-confine/ns-support.c

```diff
@@ -611,7 +611,7 @@ static void helper_main(struct sc_mount_ns *group, struct sc_apparmor *apparmor,
 		switch (errno) {
 		case ESRCH:
 			debug("parent process has terminated");
-			abort();
+			exit(0);
```

Comment:
Shouldn't this be `exit(<non-zero>)`?

Reply:
I'm not sure ... we don't know the nature of the death of our parent. At the same time, apart from system services observing termination status of all the processes, nobody will see our termination code if snap-confine itself terminates.

Reply:
Maybe a small comment here that this is a very usual condition and why its ok to exit(0)?

Reply:
I think it is correct for the child to exit(0) in this case for the reason @zyga mentioned. I do like @mvo5's idea of adding a small comment.